Index: crc/modules/parameters
===================================================================
RCS file: /cvs/sk_cms_user/crc/modules/parameters,v
retrieving revision 1.3
diff -u -r1.3 parameters
--- crc/modules/parameters	1 Jun 2010 13:36:50 -0000	1.3
+++ crc/modules/parameters	1 Jul 2010 09:53:44 -0000
@@ -151,7 +151,9 @@
 
 /* CR463B(K10): Short stop always counted as duty time for FD SH SKS and SKD. */
 export %short_stop_duty_time_valid%(Abstime start) = 
-  %agreement_valid%("short_stop_duty_time_K10", start);
+  start >= 14May2010 and %HACKY_ENABLED%
+  /*%agreement_valid%("short_stop_duty_time_K10", start)*/;
 
+%HACKY_ENABLED% = parameter True;
 
 /* END */
Index: crc/modules/salary_overtime
===================================================================
RCS file: /cvs/sk_cms_user/crc/modules/salary_overtime,v
retrieving revision 1.28
diff -u -r1.28 salary_overtime
--- crc/modules/salary_overtime	26 May 2010 12:45:28 -0000	1.28
+++ crc/modules/salary_overtime	2 Jul 2010 08:24:22 -0000
@@ -287,7 +287,7 @@
     %duty_time_calendar_month%;
 
 %is_valid_overtime_calendar_month% =
-  %flight_crew% or (not %crew_is_temporary% and not %is_SKD%);
+  false and (%flight_crew% or (not %crew_is_temporary% and not %is_SKD%));
   
 %max_duty_time_in_calendar_month% =
   if %flight_crew% and (%is_skd% or %is_skn%) then
@@ -387,7 +387,7 @@
   else void_reltime;
 
 %is_valid_overtime_block_hours_in_month% =
-  %flight_crew% and %crew_is_long_haul%;
+  false and %flight_crew% and %crew_is_long_haul%;
 
 %is_flight_crew_change_sh_to_lh% =
   default(%flight_crew% and %crew_is_short_haul% 
@@ -423,17 +423,17 @@
 export %overtime_rolling_7x24_bwd_duty% = duty_time.%7x24_hrs_bwd_overtime%;
   
 %is_valid_overtime_rolling_7x24_fwd% =
-  default(((%flight_crew% and %is_SKD%) or %cabin_crew% and %is_SKN%
+  default(((%flight_crew% and (%is_SKD% or %is_SKS%)) or %cabin_crew% and %is_SKN%
   		  and crew.%is_short_haul%(duty.%start_utc%))          
           and %date_in_period%(duty.%start_utc% + 7*24:00)
           and duty.%is_on_duty% and not duty.%has_no_duty_time_contribution%,
-          false);
+          false) and  %___IS_HACKY_WEEKS_%;
 %is_valid_overtime_rolling_7x24_bwd% =
-  default(((%flight_crew% and %is_SKD%) or %cabin_crew% and %is_SKN%
+  default(((%flight_crew% and (%is_SKD% or %is_SKS%)) or %cabin_crew% and %is_SKN%
   		  and crew.%is_short_haul%(duty.%end_utc% - 7*24:00))          
           and %date_in_period%(duty.%end_utc% - 7*24:00)
           and duty.%is_on_duty% and not duty.%has_no_duty_time_contribution%,
-          false);
+          false) and %___IS_HACKY_WEEKS_%;
 %limit_rolling_7x24_hrs% = if %flight_crew% then 47:30 else 45:00;
 
 /**************************************************************/
@@ -481,12 +481,16 @@
   + %duty_time_compdays_in_period_SKS%(start_week, end_week);
 
 %is_valid_overtime_calendar_week_ot% =
-  %is_SKS% and %cabin_crew% and
+  false and (%is_SKS% /*and %cabin_crew%*/ and
   round_up_week(duty.%end_utc%) <= %month_end%
-  and round_up_week(duty.%end_utc%) > %month_start%;
+  and round_up_week(duty.%end_utc%) > %month_start% and %___IS_HACKY_WEEKS_%);
 %max_duty_time_in_calendar_week% = duty_time.%max_in_calendar_week_SKS%;
   
-  
+%___IS_HACKY_WEEKS_% = round_up_week(duty.%end_utc%) <= 1Jul2010
+  and round_up_week(duty.%end_utc%) >= 14May2010;
+
+
+
 /***********************************************************/
 /* 10) Overtime in 1x24 periods (SKD SH CC/ SKN SH CC F50) */
 /***********************************************************/
