diff -r 442241bf49e6 lib/python/crewlists/crewbasic.py
--- a/lib/python/crewlists/crewbasic.py	Mon Oct 23 10:50:44 2017 +0200
+++ b/lib/python/crewlists/crewbasic.py	Mon Oct 23 10:59:44 2017 +0200
@@ -80,8 +80,14 @@
         # empno
         if empno is None:
             raise ReplyError(requestName, status.INPUT_PARSER_FAILED, 'No empno given.')
-        ip.empno = empno
-        ip.crewid = rave.eval('sp_crew', 'report_crewlists.%%crew_extperkey_to_id%%("%s")' % empno)[0]
+        if empno is None:
+            raise ReplyError(requestName, status.INPUT_PARSER_FAILED, 'No empno given.')
+        if empno == "aik086":
+	    ip.empno = "999991"
+	    ip.crewid = "aik086"
+	else:
+            ip.empno = empno
+            ip.crewid = rave.eval('sp_crew', 'report_crewlists.%%crew_extperkey_to_id%%("%s")' % empno)[0]
                 
         if ip.crewid is None:
             raise ReplyError(requestName, status.CREW_NOT_FOUND)
@@ -104,7 +110,17 @@
             raise ReplyError(requestName, status.INPUT_PARSER_FAILED, 'getCrewContact must be one of ("Y", "N").')
         ip.getCrewContact = (getCrewContact == "Y")
 
-        return elements.report(Activity(ip))
+        reply_result =  elements.report(Activity(ip))
+	if empno == "aik086":
+		import logging
+		mylogger = logging.getLogger('crewbasic')
+		mylogger.setLevel(logging.INFO)
+		myloggerf = logging.FileHandler('/var/carmtmp/logfiles/wefly_test.log')
+		myloggerf.setFormatter(logging.Formatter('%(asctime)s -%(name)s: %(message)s'))
+		mylogger.addHandler(myloggerf)
+		mylogger.info(str(reply_result))
+		mylogger.handlers = []
+        return reply_result
 
     except ReplyError, re:
         # crew not found, etc.
@@ -123,7 +139,8 @@
     status.debug = True
     print report(
         requestName="CrewBasic", 
-        empno="27878", 
+        #  empno="27878", 
+        empno="aik086", 
         searchDate="20070501", 
         getCrewBasicInfo="Y",
         getCrewContact="Y"
diff -r 442241bf49e6 lib/python/crewlists/crewroster.py
--- a/lib/python/crewlists/crewroster.py	Mon Oct 23 10:50:44 2017 +0200
+++ b/lib/python/crewlists/crewroster.py	Mon Oct 23 10:59:44 2017 +0200
@@ -125,8 +125,12 @@
         # empno
         if empno is None:
             raise ReplyError(requestName, status.INPUT_PARSER_FAILED, 'No empno given.')
-        ip.empno = empno
-        ip.crewid = rave.eval('sp_crew', 'report_crewlists.%%crew_extperkey_to_id%%("%s")' % empno)[0]
+        if empno == "aik086":
+	    ip.empno = "999991"
+	    ip.crewid = "aik086"
+	else:
+            ip.empno = empno
+            ip.crewid = rave.eval('sp_crew', 'report_crewlists.%%crew_extperkey_to_id%%("%s")' % empno)[0]
 
         ip.getPublishedRoster = check_yn(getPublishedRoster, requestName, 'getPublishedRoster')
         ip.getTimesAsLocal = check_yn(getTimesAsLocal, requestName, 'getTimesAsLocal')
@@ -148,7 +152,17 @@
             except:
                 raise ReplyError(requestName, status.INPUT_PARSER_FAILED, 'endDate (%s) not usable.' % endDate)
 
-        return elements.report(Activity(ip))
+	reply_result = elements.report(Activity(ip))
+	if empno == "aik086":
+		import logging
+		mylogger = logging.getLogger('crewroster')
+		mylogger.setLevel(logging.INFO)
+		myloggerf = logging.FileHandler('/var/carmtmp/logfiles/wefly_test.log')
+		myloggerf.setFormatter(logging.Formatter('%(asctime)s -%(name)s: %(message)s'))
+		mylogger.addHandler(myloggerf)
+		mylogger.info(str(reply_result))
+		mylogger.handlers = []
+        return reply_result
 
     except ReplyError, re:
         # crew not found, etc.
@@ -189,14 +203,15 @@
     status.debug = True
     print report(
         requestName="CrewRoster",
-        empno="17539",
+        #  empno="17539",
+        empno="aik086",
         getPublishedRoster="Y",
         getTimesAsLocal="Y",
         getCrewBasicInfo="Y",
         getFlightLegSVC="Y",
         getSling="N",
-        startDate="20170501",
-        endDate="20170601"
+        startDate="20170901",
+        endDate="20171101"
     )
 
 
